### PR TITLE
actions: greet: Update the PR greeting text

### LINKED ISSUE #59741: Update new contributor message to directly describe PR escalation
The welcome message points new contributors to https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html

However, the PR escalation information is found buried in this doc here: https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#pr-review-escalation

Update the new contributor message to also link directly to PR escalation, or potentially have standalone text included directly in the new contributor message.

### PR DESCRIPTION
**Pull request by carlescufi**
Update the text so it points to useful parts of the documentation regarding CI and contribution guidelines.

Also invite users to join Discord to ask for help.

Fixes #59741

### CODE REVIEW

File: .github/workflows/greet_first_time_contributor.yml
Comment (kartben) [inline .github/workflows/greet_first_time_contributor.yml:40]: nit: make the text breathe:

```suggestion
            and update your pull request if necessary.

            If you are stuck or need help please join us on [Discord](https://chat.zephyrproject.org/)
```

File: .github/workflows/greet_first_time_contributor.yml
Comment (kartben) [inline .github/workflows/greet_first_time_contributor.yml:37]: "You might also want to take this opportunity" used to be in the context of "while you're waiting for the results of CI", but not sure it really works now (or even before TBH :) ). Maybe just:

```suggestion
            If you haven't already, please make sure to review the project's [Contributor
```

File: .github/workflows/greet_first_time_contributor.yml
Comment (kartben) [inline .github/workflows/greet_first_time_contributor.yml:33]: I wonder if it's worth highlighting that "updating" would be in the form of amending/force-pushing? It might be difficult to keep this extra-bit of information concise, but there are just too many newcomers who get bitten by this.

**Review by kartben — COMMENTED** [review]
Thanks!! See my comments which I'm not sure any of them qualify as blocking. 

However, one additional thing would be to see if it's worth trying to address #59741 with this PR? Maybe at the very end "If you are stuck or need help please join us on [Discord](https://chat.zephyrproject.org/) and ask your question there. 😊 You may use the #pr-help channel in case your PR hasn't had any attention from maintainers for at least two weeks."

File: .github/workflows/greet_first_time_contributor.yml
Comment (carlescufi) [reply .github/workflows/greet_first_time_contributor.yml:33]: I tried, see what you think about it.

**Comment by carlescufi** [comment]
> However, one additional thing would be to see if it's worth trying to address #59741 with this PR? Maybe at the very end "If you are stuck or need help please join us on [Discord](https://chat.zephyrproject.org/) and ask your question there. 😊 You may use the #pr-help channel in case your PR hasn't had any attention from maintainers for at least two weeks."

I addressed #59741 but I kept the Discord text as it was. This is because Discord is not only for escalation, often it is used to get help on formatting and workflow.

**Review by kartben — APPROVED** [review]
Perfect! :)

**Review by fabiobaltieri — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]
Thanks!

**Pull request merged**
